src/FunsAnalysis.ml

```diff
@@ -104,11 +98,10 @@ let analyze_module (m : llbc_module) (funs_map : fun_decl FunDeclId.Map.t)
           (* Opaque function *)
           obj#may_fail true;
           stateful := use_state
-      | Some body -> obj#visit_statement () body.body);
+      | Some body -> obj#visit_statement () body.body)
       (* We ignore on purpose functions that cannot fail: the result of the analysis
        * is not used yet to adjust the translation so that the functions which
        * syntactically can't fail don't use an error monad. *)
-      can_fail := not f.is_global_body
```

Comment:
I'm actually a bit surprised: I think you should just put `can_fail := true` to ignore the result of the analysis (which we don't use for now) ^^'

Reply:
Ah yes I remember ! I'll just do that yes ^^